schema/sqitch.plan

```diff
@@ -299,3 +299,4 @@ computed_columns/form_change_as_project_attachment [tables/attachment tables/for
 mutations/update_milestone_form_change [mutations/update_milestone_form_change@1.7.0] 2023-05-10T23:41:13Z Sepehr Sobhani <sepehr.sobhani@gov.bc.ca> # Remove total eligible expenses from the form data when milestone type is not General
 mutations/update_form_change [mutations/update_form_change@1.6.0] 2023-04-19T23:31:07Z Gurjeet Matharu <gurjeet@matharuco.com> # fix in update_form_change to handle a null operation for archived forms.  Should now set to update.
 @1.8.0 2023-05-16T21:39:16Z Dylan Leard <dylan@button.is> # release v1.8.0
+tables/project_type_001 2023-05-11T21:32:24Z Josh Larouche <joshua.larouche@gov.bc.ca> # adding project type 001
```

Comment:
This comment could be a little more descriptive, maybe something along the lines of this example one:

`tables/report_type_003 2023-01-31T00:01:02Z Thomas Mastromonaco <thomas@button.is> # Add summary report to the report type table`